README.md

```diff
@@ -103,6 +103,23 @@ To use the `SidekiqPublisher`, this can be replaced by including
 `SidekiqPublisher::Worker`. The usual `perform_async`, etc methods will be
 available on the class but jobs will be staged in the Postgres table.
 
+### Tying to a transaction
+Jobs that are set up to use `sidekiq_publisher` will stage new
+instances to postgres within the currently active transaction. To
+guarantee that your job is run when there's a change to the system
+of record, publish it during the transaction representing that
+change. Usually, that can be accomplished by publishing in one of
```

Comment:
What are the exceptions to the "Usually"?

Reply:
For instance, if you wanted to tie a job to particular sort of change that included updates across models, it strikes me as clearer to do the publishing within an explicitly set up transaction context. The models could be changed and the job published in one place, making it clear that they're tied in that case.